Formula/openssh.rb

```diff
@@ -34,6 +34,9 @@ class Openssh < Formula
   def install
     ENV.append "CPPFLAGS", "-D__APPLE_SANDBOX_NAMED_EXTERNAL__"
 
+    # Ensure sandbox profile prefix is correct.
+    inreplace "sandbox-darwin.c", "@PREFIX@/share/openssh", etc/"ssh"
```

Comment:
Is this worth an upstream report for why `--sysconfdir=#{etc}/ssh` isn't respected here (or is it a patch issue)?